samples/notebooks/dotnet/6-memory-and-embeddings.ipynb

```diff
@@ -253,10 +253,10 @@
     }
    },
    "source": [
-    "This is done by using the `TextMemorySkill` which exposes the `recall` native function.\n",
+    "This is done by using the `TextMemorySkill` which exposes the `recallidea` native function.\n",
     "\n",
-    "`recall` takes an input ask and performs a similarity search on the contents that have\n",
-    "been embedded in the Memory Store and returns the most relevant memory. "
+    "`recallidea` takes an input ask and performs a similarity search on the contents that have\n",
```

Comment:
Should we update the sample prompts to use this as well (mainly thinking about ContextQuery)?

Reply:
Probably, but I think that should be done in another PR